vars/jepsenPipeline.groovy

```diff
@@ -187,19 +187,19 @@ def call(Map pipelineParams) {
                     }
                 }
             }
-            stage('Clean SCT Runners') {
-                steps {
-                    catchError(stageResult: 'FAILURE') {
-                        script {
-                            wrap([$class: 'BuildUser']) {
-                                dir('scylla-cluster-tests') {
-                                    cleanSctRunners(params, currentBuild)
-                                }
-                            }
-                        }
-                    }
-                }
-            }
+//             stage('Clean SCT Runners') {
```

Comment:
Please, add a reason why it commented out.

Reply:
Fixed: I added a comment for the commented out block